src/libs/common/middleware/errorHandler.ts

```diff
@@ -1,10 +0,0 @@
-import { NextFunction, Request, Response } from 'express';
-
-export const errorHandler = (err: any, req: Request, res: Response, next: NextFunction) => {
-    console.error(err.stack);
-
-    res.status(err.status || 500).json({
-        message: err.message || 'Something went wrong',
-        error: err.message || 'Internal Server Error',
```

Comment:
Usually the error represents the error code, like INTERNAL_SERVER_ERROR, FORBIDDEN, UNAUTHORIZED, ..etc.